coralproject/talk
# [CORL-1700] Live Chat: reply notifications + reply tweaks

## What does this PR do?

- Highlight replies to the viewer's comments in the live chat
- Swap "reply" for "view conversation" on the reply comments in the main live chat stream
- When a user selects a reply comment and clicks "view conversation" the conversation thread opens and does the following:
   - jumps to the reply comment they clicked on to open the conversation
   - highlights the reply comment so they have context and focus based on the thread reply they clicked on

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Go to live chat
- Create a comment (original commenter)
- Use another account (replier) to reply to that comment
- See that the reply is highlighted with an emphasized dark grey L-bracket when it comes in on the original commenter account
- Create some more replies to the original comment
- Click on the replies new "view conversation" button and see that the conversation view shows the comment you came from as well as highlights it

Some screenies:
<img width="415" alt="image" src="https://user-images.githubusercontent.com/5751504/116126323-bd318d80-a683-11eb-890a-061faf53c819.png">

<img width="479" alt="image" src="https://user-images.githubusercontent.com/5751504/116126350-c9b5e600-a683-11eb-92d4-b02ab5500ae9.png">
